### ISSUE TITLE
Increase contrast ratios in color theme

### ISSUE THREAD

**Issue by nbudin** [neinteractiveliterature/intercode#1945]
Many of the default Bootstrap theme colors lead to inaccessible elements in terms of color contrast ratio.  WCAG AA recommendation is at least a 4.5:1 contrast ratio between foreground and background colors.  We're sitting at around 3.9:1 for default links on white text right now.

If we come up with some reasonable defaults, we could release a Github repo containing our recommendations so that others could easily take advantage of it too.

**Comment by lisefrac** [comment]
Sticking this here for my own reference: https://webaim.org/resources/contrastchecker/

**Comment by lisefrac** [comment]
So, I'm looking at the U site on my local as an example:

- The default link color is #007bff, which I'm assuming is the default link color @nbudin is referring to, since it gives the same ratio he mentions. 
- By lowering the blue digits from "ff" to "bb" (#007bbb, option 1) we can get up to 4.61:1 contrast, which meets WCAG standards. "#007baa" (option 2) will get us 4.75:1, if we feel like being overachievers. 
- We can also mess with the green bits to get "#002baa" (option 3) which gives us a whopping 10.9:1 contrast, but then the link starts to look similar to the copy, which is a different contrast problem (is that a contrast problem we want to solve here? It doesn't quite seem like the same problem)
- Either way, the default link color is set in _reboot.css, which appears to be a fork of normalize.css, so I assume we would want to fork it even further with the WCAG-compliant colors. 

Before I make these changes, does anyone want to weigh in? (Or should I just use my best judgment?) Please see screenshots below:

<img width="769" alt="option1-007bbb" src="https://user-images.githubusercontent.com/88073/89128069-f8ba7500-d4c0-11ea-9df9-f7a58b8f56d9.png">
Option 1

<img width="753" alt="option2-007baa" src="https://user-images.githubusercontent.com/88073/89128072-fb1ccf00-d4c0-11ea-850f-df29a0e05197.png">
Option 2

<img width="749" alt="option3-002baa" src="https://user-images.githubusercontent.com/88073/89128075-ff48ec80-d4c0-11ea-9f1d-48f8cd2d7837.png">
Option 3

**Comment by lisefrac** [comment]
gah, and I totally named these wrong; please hold

**Comment by lisefrac** [comment]
Okay, screenshots should be correct now. Let me know if you have an opinion, Intercode team :)

**Comment by lisefrac** [comment]
Tagging @marleighnorton

**Comment by nbudin** [comment]
I definitely want to hear from others as well, but both options 1 and 2 look nice to me.  I agree Option 3 is a little too similar to the plain text color, and also the hue shift in it bothers me a bit.

**Comment by marleighnorton** [comment]
For me, all the link colors are clear enough, but I wonder if color blind folks would have issues.  Either looking too similar to the text or too similar to the grey in "Ultraviolet."  Hmm.

Is hover underline still a thing for links, or am I stuck in the 90s?

**Comment by lisefrac** [comment]
I believe those links do have an underline on hover — but I’ll double check when I’m back at my computer. (I should have thought to hover when I took my screenshots!) 

@natbudin, do happen to know if the WCAG contrast guidelines take into account color blindness? I sorta assumed they did, but I don’t know enough about how color-blindness works. (Although I believe it’s usually keyed to red-green, and we’re only changing the blue bits of the hex value, except for the last option).

Worth noting, we could also make the links stand out more by bolding them. Maybe not to be addressed in this ticket.

Also worth noting, I think the hover color is adaptive based on the color of the link. So I’ll want to check the contrast there, too, before we call this done.

**Comment by nbudin** [comment]
I've just taken some colorblindness-simulated screenshots using Color Oracle.  Here are the results:

| Deuteranopia | Protanopia |
|:-------------|:-----------|
| ![Deuteranopia](https://user-images.githubusercontent.com/7477/89207773-20b4e180-d570-11ea-8ee0-2024deddbd3a.png) | ![Protanopia](https://user-images.githubusercontent.com/7477/89207778-23173b80-d570-11ea-9c09-9ff042548886.png) |
| Tritanopia | Grayscale |
| ![Tritanopia](https://user-images.githubusercontent.com/7477/89207783-26122c00-d570-11ea-962a-387d6712c804.png) | ![Grayscale](https://user-images.githubusercontent.com/7477/89207789-27dbef80-d570-11ea-87c5-7eced60eab2f.png) |

Looking at these, I feel like Options 1 and 2 are both reasonable with any of these filters, but Option 3 is a little difficult to distinguish from the body text particularly in grayscale.

**Comment by lisefrac** [comment]
Nice research, @nbudin !

Given that, I think I will probably move ahead with Option 1, unless anyone wants to weigh in. It seems slightly better than 2 under the greyscale and protanopia filters, I think.

**Comment by marleighnorton** [comment]
Works for me.  Option 1 it is!

**Comment by nbudin** [comment]
Since we upgraded to Bootstrap 5, this has now actually been fixed for us by the framework!  🎉
